src/app/subscriptions/keyboard_shortcuts.subscriptions.ts

```diff
@@ -0,0 +1,28 @@
+import buildKeyCombination from '../actions/shortcuts.actions';
+
+// KeyDown Subscription
+const subKeyDown = (dispatch, props) => {
+  const hander = event => {
+    dispatch([props.action, event]);
```

Comment:
```suggestion
    dispatch(props.action, event);
```

Avoids additional `dispatch` call inside hyperapp's code